rome-modules/src/main/java/com/rometools/modules/atom/modules/AtomLinkModuleImpl.java

```diff
@@ -49,32 +52,24 @@ public String getUri() {
     @Override
     public void copyFrom(CopyFrom obj) {
         AtomLinkModule other = (AtomLinkModule) obj;
-        Link link = other.getLink();
-        if (link != null) {
+        List<Link> links = other.getLinks();
+        for (Link link : links) {
             Link l = new Link();
             l.setHref(link.getHref());
             l.setType(link.getType());
             l.setRel(link.getRel());
             l.setHreflang(link.getHreflang());
             l.setTitle(link.getTitle());
             l.setLength(link.getLength());
-            setLink(l);
+            links.add(l);
         }
     }
 
     @Override
     public Object clone() {
         final AtomLinkModuleImpl m = new AtomLinkModuleImpl();
-        if (link != null) {
-            Link l = new Link();
-            l.setHref(link.getHref());
-            l.setType(link.getType());
-            l.setRel(link.getRel());
-            l.setHreflang(link.getHreflang());
-            l.setTitle(link.getTitle());
-            l.setLength(link.getLength());
-            m.setLink(l);
-        }
+        List<Link> result = links.subList(0, links.size());
```

Comment:
This code doesn't clone each link anymore. Not sure if it matters, but is it intentional?